src/common/scrub_types.cc

```diff
@@ -240,6 +253,19 @@ void inconsistent_snapset_wrapper::encode(bufferlist& bl) const
   ENCODE_FINISH(bl);
 }
 
+bufferlist inconsistent_snapset_wrapper::encode() const
```

Comment:
Have this one call the out-param version.  I *really* don't want two copies of encode.